securitycenter/notifications/create_notification_config.go

```diff
@@ -0,0 +1,49 @@
+package notifications
+
+// [START scc_create_notification_config]
+import (
+	"context"
+	"fmt"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv1p1beta1"
+	securitycenterpb "google.golang.org/genproto/googleapis/cloud/securitycenter/v1p1beta1"
+)
+
+func createNotificationConfig(orgId string, pubsubTopic string, notificationConfigId string) error {
```

Comment:
```suggestion
func createNotificationConfig(orgID string, pubsubTopic string, notificationConfigID string) error {
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done